scripts/fetch-ui-assets.sh

```diff
@@ -12,6 +12,6 @@
 # respective releases in "influxdata/ui" (OSS-2.0, OSS-2.1, etc). Those releases
```

Comment:
I moved this file in the `scripts` directory. I didn't want to end up with anything in the `ui` directory other than the `README.md`.